tests/unit-tests/gconstruct/test_construct_graph.py

```diff
@@ -1701,10 +1701,63 @@ def test_feature_wrapper():
 
 
 def test_gc():
-    assert not os.path.isdir("/tmp_featurewrapper"), \
-        "Directory /tmp_featurewrapper should not exist after gc"
-    assert not os.path.isdir("/tmp_featurewrapper2"), \
-        "Directory /tmp_featurewrapper2 should not exist after gc"
+    assert not os.path.isdir("/tmp/featurewrapper"), \
+        "Directory /tmp/featurewrapper should not exist after gc"
+    assert not os.path.isdir("/tmp/featurewrapper2"), \
+        "Directory /tmp/featurewrapper2 should not exist after gc"
+
+
+def test_homogeneous():
```

Comment:
What is the purpose this this test?
Does the test belong to this PR?

Reply:
I think was added during the conflict resolution, I'll remove it.

Reply:
@jalencato Do you know what test is it?